packages/fccsw/package.py

```diff
@@ -42,12 +42,12 @@ class Fccsw(CMakePackage):
     depends_on('geant4', when='@0.13:')
     depends_on('geant4@10.6.1', when='@:0.12')
     depends_on('hepmc@:2.99.99')
-    depends_on('heppdt@:2.99.99')
```

Comment:
Why this change? I recall getting a build error with the 3.X version, and all the lcg releases use 2.06.01.